.claude/workflows/security-hardening.yaml

```
@@ -0,0 +1,200 @@
+  owasp_top_10:
+    - injection_flaws
+    - broken_authentication
+    - sensitive_data_exposure
+    - xml_external_entities
+    - broken_access_control
+    - security_misconfiguration
+    - cross_site_scripting
+    - insecure_deserialization
+    - vulnerable_components
+    - insufficient_logging
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)